packages/tfm-ror51/tfm-ror51.spec

```diff
@@ -0,0 +1,162 @@
+%global scl_name_prefix tfm-
+%global scl_name_base ror
+%global scl_name_version 51
+
+%global scl %{scl_name_prefix}%{scl_name_base}%{scl_name_version}
+
+# Fallback to rh-ruby24. rh-ruby24-scldevel is probably not available in
+# the buildroot.
+%{!?scl_ruby:%global scl_ruby rh-ruby24}
+%{!?scl_prefix_ruby:%global scl_prefix_ruby %{scl_ruby}-}
+
+# Do not produce empty debuginfo package.
+%global debug_package %{nil}
+
+# Support SCL over NFS.
+# nfsmountable macro must be defined before defining the scl_package macro
+%global nfsmountable 1
+
+%{!?install_scl: %global install_scl 1}
+
+%scl_package %scl
+
+Summary: Package that installs %scl
+Name:    %scl_name
+Version: 1.0
+Release: 1%{?dist}
+License: GPLv2+
+Source0: README
+Source1: LICENSE
+
+%if 0%{?install_scl}
```

Comment:
I'm not sure what to make of this section. What use case is there for ```install_scl``` set to 0?

Reply:
I think this is whether or not you'd want to have this RPM install the SCL requirements. As in treat this RPM as the master of the SCL which then installs all of the SCL packages. I think we can drop it too.